addon/styles/addon.scss

```diff
@@ -44,6 +44,12 @@ $frost-modal-overflow-shadow-color: rgba(0, 0, 0, .13);
   }
 }
 
+.form {
```

Comment:
Do we need this selector?  The only occurrence of the `frost-modal-dialog-header` class in all of `ciena-frost` is to this component (https://github.com/search?q=org%3Aciena-frost+frost-modal-dialog-header&type=Code) and the `.frost-modal-dialog` selector/definition is not targeted the same way.

Reply:
Ah ur right I was using a default class modifier